scripts/dual_iir_configuration.py

```diff
@@ -188,7 +188,7 @@ def calculate_pid_coefficients(sampling_period, args):
         [1, -2, 1]
     ]
 
-    digital_conversion = (2 * pi) / sampling_period
+    digital_conversion = 2 * pi * sampling_period
```

Comment:
I'm confused. I thought that `omega_s = 2 pi / T`.

Is this conversion type not actually `omaga_s`?

Reply:
In the definitions of the primer, `kd*2*pi` is the D gain at the sample frequency. Or `ki/tau` is the I gain "slope". Then
`kd*2*pi*f/fs` and `ki/(2*pi*f/fs)` are gains at f.

Reply:
Ah gotcha. That makes more sense, thanks for the clarification. This is a scaling factor to convert some given `f` input.